packages/core/helper-plugin/src/hooks/useSelectionState/index.js

```diff
@@ -34,5 +34,31 @@ export const useSelectionState = (keys, initialValue) => {
     }
   };
 
-  return [selections, { selectOne, selectAll, selectOnly, setSelections }];
+  const selectMultiple = (nextSelections) => {
+    setSelections([
+      ...selections,
+      ...nextSelections.filter(
+        (nextSelection) =>
+          selections.findIndex((currentSelection) =>
+            keys.every((key) => currentSelection[key] === nextSelection[key])
+          ) === -1
+      ),
+    ]);
+  };
+
+  const deselectMultiple = (nextSelections) => {
+    setSelections([
```

Comment:
I would use the function format of `setSelections` – `setSelections(currSelections => ...` this is more inline with the rest of the hook.

Reply:
Thanks for the advice. I absolutely forgot about that possibility and it definitely makes it much more readable